build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
What about making a `Makefile` ? :thinking:

Reply:
First of all, I would say that these files are generated by the dotnet template; I didn't write it 😉 

Then, why not using a Makefile, right?
To me the main advantage to use a Makefile as:
```
target: dependencies
    command
```
is that the target will be built incrementally when its dependencies have changed.
But, my knowledges of the `dotnet` ecosystem is too poor to write a Makefile: I don't really know what produces and what are the dependencies of a `dotnet restore` and a `dotnet build` commands 🤷‍♂️
But don't hesitate to make a concret proposition 😉

Reply:
From what I see, we need 2 targets
I'd suggest:

```suggestion
restore: ## Restore project dependencies
    dotnet restore src/Releasy

build: restore ## Build the whole project
    dotnet build src/Releasy

help:
    @grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'

.DEFAULT_GOAL := help
.PHONY: restore build help
```

Reply:
If you need build, can I test gradle on it?

Reply:
Of course, everybody can test what he or she wants 😉 And if the experiment is conclusive, why not to integrate it.
But, for now, I'm not sure I want to depend upon a new external tool. Moreover, I would like to be close enough to what is used in the Dockerfile.
And don't forget that, at this stage of the project, this build command run in about 5 sec. I'm not sure in the main painful point 💁‍♂️

Reply:
Fix in 851ae2c746bf54a2af4425bf8eb4450e656b6e84